tests/unit/plugins/filter/test_find_template_roles.py

```
@@ -0,0 +1,288 @@
+                return Metadata.model_validate(yaml.safe_load(f))
+    pytest.fail(f"osac.yaml not found for role {role_name}")
+
+
+# ---------------------------------------------------------------------------
+# TestTemplateParameterDefinitionDefaults
+# ---------------------------------------------------------------------------
+
+class TestTemplateParameterDefinitionDefaults:
+
+    @pytest.mark.parametrize(
+        "value, expected_type",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)